sql/moz-fx-data-shared-prod/telemetry_derived/experiment_enrollment_cumulative_population_estimate_v1/query.sql

```diff
@@ -0,0 +1,149 @@
+WITH all_enrollments AS (
```

Comment:
I did some performance testing here. Instead of querying the view when exporting the data to GCS, it is more than 5x faster if the data is persisted in a table that is queried by the export script. So I changed this into a query.